### DISCUSSION TITLE
Short question.. Start the StartMenu.exe with swiches.. i wana directly open ..

### DISCUSSION THREAD

**Discussion by blackcrack** [Open-Shell/Open-Shell-Menu#801] [Q&A] [answered]
Hi,

i know it works, and it is in the wiki, but i don't know where directly,
i have look in the github wiki, but there find i not directly the switch
for open it directly.. , do anyone know, with switch is it who open the startmenu directly ..

to use in Total Commander or rocketdock as icon StartMenu.exe with switch for open directly

thy best
blacky

**Comment by plofhaan** [comment, marked as answer]
"C:\Program Files\Open-Shell\StartMenu.exe" -open

**Reply by blackcrack** [reply]
> 
> 
> "C:\Program Files\Open-Shell\StartMenu.exe" -open

thank you :)

**Reply by plofhaan** [reply]
You are welcome, I found it here: http://www.classicshell.net/forum/viewtopic.php?f=7&t=1584 :)
